doc/persistence.rst

```diff
@@ -13,35 +13,35 @@ Persistence
 
 .. currentmodule:: joblib.numpy_pickle
 
-Usecase
-=======
+Use case
+========
 
 :func:`joblib.dump` and :func:`joblib.load` provide a replacement for
-pickle to work efficiently on Python objects containing large data, in
-particular large numpy arrays.
+pickle to work efficiently on arbitrary Python objects containing large data,
+in particular large numpy arrays.
 
 A simple example
 ================
 
-First we create a temporary directory::
+First create a temporary directory::
 
   >>> from tempfile import mkdtemp
   >>> savedir = mkdtemp()
   >>> import os
   >>> filename = os.path.join(savedir, 'test.pkl')
 
-Then we create an object to be persisted::
+Then create an object to be persisted::
 
   >>> import numpy as np
   >>> to_persist = [('a', [1, 2, 3]), ('b', np.arange(10))]
 
-which we save into `savedir`::
```

Comment:
using `savedir` makes no sense now that `dump` pickles numpy arrays in a single file ;)